src/copy-to-clipboard/index.tsx

```diff
@@ -1,98 +1,21 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import React, { useState } from 'react';
-import { getBaseProps } from '../internal/base-component';
+import React from 'react';
 import { CopyToClipboardProps } from './interfaces';
 import { applyDisplayName } from '../internal/utils/apply-display-name';
 import useBaseComponent from '../internal/hooks/use-base-component';
-import InternalButton from '../button/internal';
-import InternalPopover from '../popover/internal';
-import InternalStatusIndicator from '../status-indicator/internal';
-import styles from './styles.css.js';
-import testStyles from './test-classes/styles.css.js';
-import clsx from 'clsx';
+import InternalCopyToClipboard from './internal';
+//import { getExternalProps } from '../internal/utils/external-props';
 
 export { CopyToClipboardProps };
 
-export default function CopyToClipboard({
-  variant = 'button',
-  copyButtonAriaLabel,
-  copyButtonText,
-  copySuccessText,
-  copyErrorText,
-  textToCopy,
-  ...restProps
-}: CopyToClipboardProps) {
-  const { __internalRootRef } = useBaseComponent('CopyToClipboard', {
+export default function CopyToClipboard({ variant = 'button', ...restProps }: CopyToClipboardProps) {
+  const baseProps = useBaseComponent('CopyToClipboard', {
     props: { variant },
   });
-  const baseProps = getBaseProps(restProps);
+  //const filteredProps = getExternalProps(restProps);
```

Comment:
Why is this line commented? It should be used

```js
const filteredProps = getExternalProps(restProps);

return <InternalCopyToClipboard variant={variant} {...baseProps} {...filteredProps} />;

```

Reply:
All good and ready to merge otherwise

Reply:
I commented that code out since some tests were failing, but I tried it again with the code uncommented, and it still pass all the tests.